homeassistant/components/onewire/const.py

```diff
@@ -66,3 +66,6 @@
     SENSOR_DOMAIN,
     SWITCH_DOMAIN,
 ]
+
+MICROLAN_TYPE = "DS2409"
+MICROLAN_BRANCHES = ["aux", "main"]
```

Comment:
To follow the other modules, could you instead create a local constant on `onewirehub.py` like this?

```
DEVICE_COUPLERS = {
    # Family : [branches]
    "1F": ["aux", "main"]
}
```

Reply:
Thanks for your help, and for your suggestions - recursion into `_discover_devices_owserver()` is much cleaner!  I have updated the pull request.